### PR TITLE
Add automatic creation of elasticsearch indices and policies after ap…

### PR DESCRIPTION
**Pull request by bmonkman**
…plying. 
ES cluster is internal so this has to be done via a container in k8s.

It would be nice to have this tied in to the terraform but the user running TF won't be able to reach the ES cluster.

### CODE REVIEW

File: templates/scripts/elasticsearch-logging.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
+# This file wil run locally. It will run a run a job inside the kubernetes cluster that will set up indices and a lifecycle policy in Elasticsearch
+
+DOCKER_IMAGE_TAG=commitdev/zero-k8s-utilities:0.0.2
+ES_ENDPOINT=$(aws es describe-elasticsearch-domain --domain-name bill-aug21-stage-logging --query "DomainStatus.Endpoints.vpc" | jq -r '.')
```
Comment (davidcheung) [inline templates/scripts/elasticsearch-logging.sh:5]: need to pass in variables from makefile 
https://github.com/commitdev/zero-deployable-backend/blob/master/Makefile#L21-L25

```suggestion
ES_ENDPOINT=$(aws es describe-elasticsearch-domain --domain-name $PROJECT-$ENVIRONMENT-logging --query "DomainStatus.Endpoints.vpc" | jq -r '.')
```

File: templates/terraform/README.md [hunk 2]
```diff
@@ -131,3 +131,39 @@ The process should be:
...
 Done!
+
+<% if eq (index .Params `loggingType`) "kibana" %>
+## Kibana and Elasticsearch index Management
+
+After creating the AWS Elasticsearch cluster to hold log data it’s a good idea to create index policies to control how data ages over time.
+
+Typically you will want different policies on Staging and Production, as staging will probably have less restrictions about availability and speed, and more retained data increases cost.
+
+You can view these in Kibana's Index Management UI by clicking on the "IM" tab, but some default indices and lifecycles are automatically created. You can see the policies that were created in [scripts/files/](scripts/files/)
```
Comment (davidcheung) [inline templates/terraform/README.md:142]:
```suggestion
You can view these in Kibana's Index Management UI by clicking on the "IM" tab, but some default indices and lifecycles are automatically created. You can see the policies that were created in [scripts/files/](../scripts/files/)
```

**Comment by bmonkman** [comment]
Just running through one more full  `zero apply` to flush out any other little issues.

**Comment by davidcheung** [comment]
nice, reminder to squash the commits since our default github behavior doesn't do that

**Comment by bmonkman** [comment]
I was actually looking for the option to require squash yesterday and couldn't find it. Just looked again and remembered where it was. Thanks!

**Pull request merged**